### ISSUE TITLE
Build fails with cmake -DBuildTests=ON

### ISSUE THREAD

**Issue by slodki** [moneymanagerex/moneymanagerex#1214]
- Check OS
  - [ ] Windows
  - [x] Mac OSX
  - [x] Linux 
- Check MMEX version
  - [x] git master

```
/usr/bin/c++   -DWXUSINGDLL -D_FILE_OFFSET_BITS=64 -D__WXGTK__ -isystem /usr/lib/x86_64-linux-gnu/wx/include/gtk2-unicode-3.0 -isystem /usr/include/wx-3.0 -I/moneymanagerex/src -I/moneymanagerex/3rd -I/moneymanagerex/3rd/cgitemplate -I/moneymanagerex/3rd/csv-parser/include -I/moneymanagerex/3rd/LuaGlue/include -I/moneymanagerex/lib/cppunit/include -I/moneymanagerex/lib/lua/src -I/moneymanagerex/lib/wxsqlite3/include -I/moneymanagerex/lib/wxsqlite3/sqlite3/secure/src  -pthread -g   -std=gnu++11 -o CMakeFiles/CPPUnit.dir/lib/cppunit/src/cppunit/AdditionalMessage.cpp.o -c /moneymanagerex/lib/cppunit/src/cppunit/AdditionalMessage.cpp
In file included from /moneymanagerex/lib/cppunit/include/cppunit/Message.h:4:0,
                 from /moneymanagerex/lib/cppunit/include/cppunit/AdditionalMessage.h:4,
                 from /moneymanagerex/lib/cppunit/src/cppunit/AdditionalMessage.cpp:1K:
/moneymanagerex/lib/cppunit/include/cppunit/Portability.h:18:37: fatal error: cppunit/config-auto.h: No such file or directory
 #    include <cppunit/config-auto.h>
                                     ^
compilation terminated.
CMakeFiles/CPPUnit.dir/build.make:65: recipe for target 'CMakeFiles/CPPUnit.dir/lib/cppunit/src/cppunit/AdditionalMessage.cpp.o' failed
make[2]: *** [CMakeFiles/CPPUnit.dir/lib/cppunit/src/cppunit/AdditionalMessage.cpp.o] Error 1
make[2]: Leaving directory '/moneymanagerex/build/release'
CMakeFiles/Makefile2:108: recipe for target 'CMakeFiles/CPPUnit.dir/all' failed
make[1]: *** [CMakeFiles/CPPUnit.dir/all] Error 2
```

I think there is a problem with cmake script for cppunit.

**Comment by slodki** [comment]
Maybe something like this (building autotools project from cmake):
```
include(ExternalProject)
ExternalProject_Add(
  mylib
  SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/lib/mylib-x.y.z
  CONFIGURE_COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/lib/mylib-x.y.z/configure --prefix=${CMAKE_CURRENT_SOURCE_DIR}/lib/mylib-x.y.z
  PREFIX ${CMAKE_CURRENT_SOURCE_DIR}/lib/mylib-x.y.z
  BUILD_COMMAND make
  BUILD_IN_SOURCE 1
)
```

**Comment by slodki** [comment]
This should be fixed with PR #1313 

Windows binary packages with `mmex-tests` included are [ready to download](https://ci.appveyor.com/project/slodki/moneymanagerex/build/968). No 64-bit packages ac CPPUnit library project file support only Win32.

Linux packages [can be build without errors](https://travis-ci.org/slodki/moneymanagerex/builds/271357186). For macOS there is [small problem](https://travis-ci.org/slodki/moneymanagerex/jobs/271357204#L2949) with `libtool` brew package running under OS X Yosemite.

To build `mmex-tests` option `-DMMEX_BUILD_TESTS=ON` must be added to cmake command.
